html/index.php

```diff
@@ -1,7 +1,72 @@
 <?php
 require '../vendor/autoload.php';
+require_once 'HTML/Table.php';
```

Comment:
Rather than depend on php extensions, could you achieve the same thing with a composer package?

Reply:
For example, it'd be relatively easy to pull in twig and use it to generate the table, removing the need to install an extension.

https://stackoverflow.com/questions/10899751/can-i-use-twig-in-standalone-project